db/migrate/20190313134642_migrate_default_user_email_options.rb

```diff
@@ -0,0 +1,31 @@
+class MigrateDefaultUserEmailOptions < ActiveRecord::Migration[5.2]
+  def up
+    email_always = DB.query_single("SELECT value FROM site_settings WHERE name = 'default_email_always'").first
+    email_direct = DB.query_single("SELECT value FROM site_settings WHERE name = 'default_email_direct'").first
+    email_personal_messages = DB.query_single("SELECT value FROM site_settings WHERE name = 'default_email_personal_messages'").first
+
+    default_email_level = nil
+    default_email_level = UserOption.email_level_types[:always] if email_direct != 'f' && email_always == 't'
```

Comment:
This is one of those cases where enums are not great. 

It is *probably* safe but we've found if an enum changes in the future (say a key is deleted) the migration breaks. It's counter intuitive but it's probably safer here if you hard code the values.